service/replication.go

```diff
@@ -113,8 +111,8 @@ func (s *service) CreateRemoteVolume(ctx context.Context,
 		var quotaID string
 		quota, err := remoteIsiConfig.isiSvc.client.GetQuotaWithPath(ctx, exportPath)
 		log.Info("Get quota", quota)
-		if quota == nil {
-			log.Info("Remote quota doesn't exists, create it")
+		if err != nil {
```

Comment:
It is possible that you may get an error for other reasons than if the quota does not exist. For example the REST call may fail for a number of reasons. May be worth refactoring this in the future to handle such cases. If the quota did exist but the REST call failed then the subsequent operation to create the quota will also fail. How does the API behave if there is a call to get a quota that does not exist, is there a 404 or similar error?